pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.nearsoft.fgaribay</groupId>
+    <artifactId>skt-task</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>skt-task</name>
+    <description>SKT Task Project</description>
+
+    <parent>
```

Comment:
As a personal preference, I avoid using this Spring Boot parent POM and use the `<dependencyManagement>` tag instead:
-  [Spring Boot Dependency Management with a Custom Parent](https://www.baeldung.com/spring-boot-dependency-management-custom-parent)

Using this approach, make sure that the `spring-boot-maven-plugin` in `management-app/pom.xml` file specifies the plugin version (`1.5.7.RELEASE` instead of `2.0.3.RELEASE`), which you can define as a property in this POM file and reuse it in the submodule.